services/user.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+User = get_user_model()
```

Comment:
According to the checklist, you should use `get_user_model()` directly instead of assigning it to `User`. This is to ensure that the most current user model is always used.